test/ffmpeg-qsv/transcode/avc.py

```diff
@@ -0,0 +1,190 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .transcoder import TranscoderTest
+
+spec = load_test_spec("avc", "transcode")
+class to_avc(TranscoderTest):
+  @slash.requires(have_ffmpeg_h264_qsv_encode)
+  @slash.requires(have_ffmpeg_h264_qsv_decode)
+  @slash.parametrize(*gen_transcode_1to1_parameters(spec, "avc", "hwhw"))
+  @platform_tags(set(AVC_DECODE_PLATFORMS) & set(AVC_ENCODE_PLATFORMS))
+  def test_hwhw_1to1(self, case):
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      dstextension = 'h264',
+      case         = case,
+      mode         = 'hwhw',
+      ffencoder    = 'h264_qsv',
+      ffdecoder    = 'h264_qsv',
```

Comment:
Please alphabetize these keys in all cases.

Reply:
added in alphabetical order